openslides_backend/action/meeting/delete_all_speakers_of_all_lists.py

```diff
@@ -0,0 +1,36 @@
+from ...models.models import Speaker
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import ActionPayload, DeleteAction
+from ..register import register_action
+
+
+@register_action("meeting.delete_all_speakers_of_all_lists")
+class DeleteAllSpeakersOfAllListsAction(DeleteAction):
+    """
+    Action to delete all speakers of all lists of a meeting.
+    """
+
+    model = Speaker()  # we want to delete Speakers
+    schema = DefaultSchema(Speaker()).get_delete_schema()
```

Comment:
This schema is not correct. It produces a missleading title and description.

Reply:
Updated the schema.